docs/discussion/Multi-Tenancy Support.md

```diff
@@ -0,0 +1,168 @@
+Multi-Tenancy Support
+==
+Author: Binbin Li (@binbin-li)
+
+# Background
+Ratify’s first stable version for production has been officially released, garnering increased adoption as an external data provider among a growing number of customers. In production environments, numerous clients are opting for the implementation of a multi-tenant architecture to enhance resource efficiency within their clusters. To support this scenario, Ratify needs to make some essential refactoring. Enclosed in this document are several proposed design approaches and their respective modifications, warranting further scrutiny and discussion.
+
+# Multi-tenancy category
+Firstly, the concept of multi-tenancy within a Kubernetes cluster can be broadly categorized into two distinct classifications: multi-team and multi-customer.
```

Comment:
Would it be better to add this link as a reference for concept details? https://kubernetes.io/docs/concepts/security/multi-tenancy/